docs/docusaurus/docs/core/connect_to_data/sql_data/sql_data.md

```diff
@@ -10,7 +10,6 @@ import TabItem from '@theme/TabItem';
 import Tabs from '@theme/Tabs';
 
 import ConfigureCredentials from '../configure_credentials/_configure_credentials.md';
-import SecretsManagers from '../configure_credentials/_secrets_managers.md'
```

Comment:
**Note for reviewer** - I'm removing this import and the associated file because they were left lingering and unused after the display of this content was removed in https://github.com/great-expectations/great_expectations/pull/10157/files#diff-ccdd013ab298d7c2d2b22d361b53f490b81db4d6281a478b552039b902c1448bL36 -